samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/MainActivity.kt

```diff
@@ -19,17 +20,21 @@ import dagger.hilt.android.AndroidEntryPoint
 import io.customer.android.sample.kotlin_compose.navigation.AppNavGraph
 import io.customer.android.sample.kotlin_compose.ui.login.AuthenticationViewModel
 import io.customer.android.sample.kotlin_compose.ui.theme.CustomerIoSDKTheme
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
 
 @AndroidEntryPoint
 class MainActivity : ComponentActivity() {
 
     private val authenticationViewModel: AuthenticationViewModel by viewModels()
+    private val deepLinkState: MutableStateFlow<Intent?> = MutableStateFlow(null)
```

Comment:
Just to make variable slightly more descriptive 

```suggestion
    private val deepLinkIntentState: MutableStateFlow<Intent?> = MutableStateFlow(null)
```